src/workerd/api/http.h

```diff
@@ -751,6 +752,8 @@ struct RequestInitializerDict {
   // jsg::Optional<kj::String> priority;
   // TODO(conform): Might support later?
 
+  void validate();
```

Comment:
nit: let's move this to after the `JSG_STRUCT` and `JSG_STRUCT_TS_OVERRIDE` macros and add a brief comment pointing folks to the new validate functionality in the `JSG_STRUCT` definition. Otherwise this might be a bit confusion for someone coming into this code fresh thinking they might need to call `validate()` themselves.